pkg/products/marin3r/rateLimitService_test.go

```diff
@@ -31,7 +31,12 @@ func TestRateLimitService(t *testing.T) {
 			Reconciler: NewRateLimitServiceReconciler(&marin3rconfig.RateLimitConfig{
 				Unit:            "minute",
 				RequestsPerUnit: 1,
-			}, "redhat-test-marin3r", "ratelimit-redis"),
+			},
+				&integreatlyv1alpha1.RHMI{
+					Spec: integreatlyv1alpha1.RHMISpec{
+						PriorityClassName: "rhoam-pod-priority",
```

Comment:
Would it make sense to make for this value to be a const instead of hard-coding it? If the value ever needs to be changed, we only need to make a change to the const value.

Reply:
Hey @obrienrobert , thanks for the review/question! 
I think it should not matter, reason why is that it is a test so we are mocking the value in the RHOAM CR and later on, comparing if the expected result matches the value above. So essentially it could be anything at all. I think that we could even live without these values being set at all in the first 3 tests, as it is only the last one that cares about this value.

Reply:
@MStokluska understood, thanks for the detailed reply!

Reply:
> I think that we could even live without these values being set at all in the first 3 tests
Just reading over this again - If these values are not needed for the first 3 tests, could they be removed?

Reply:
> I think that we could even live without these values being set at all in the first 3 tests

If we don't need this value for the first 3 tests, could it be removed?

Reply:
@obrienrobert 
Sure! Done! Sorry was meant to do it yesterday!